app/public/src/style/toastify.css

```diff
@@ -19,17 +19,4 @@
 .toast-new-pokemon img {
   width: 40px;
   height: 40px;
-  image-rendering: pixelated;
-}
-
-.toast-emote {
-  right: 85px !important;
```

Comment:
Why are those 2 selectors deleted ?

Reply:
theyre unused as far as i can tell

Reply:
ok yeah it seems the real used class is "game-emote-bubble"